src/app/(auth)/sign-up/page.tsx

```
@@ -0,0 +1,24 @@
+
+export const metadata: Metadata = {
+  title: "Mimir - Sign Up",
+  description: "Create an account to use mimir",
+};
+
+export default async function SignUpPage() {
+  const session = await auth.api.getSession({
+    headers: await headers(),
+  });
+
+  if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)